README.md

```diff
@@ -20,14 +20,22 @@ specific containers to use this project.
 ### Get started
 All your Drupal files should be placed into the folder "web".
 
+Add your env variable for dev.
+```sh
+$ cd to project root
+$ export BUILD_ENV=dev
```

Comment:
I started testing this again and I got the variable passed to the container with the .env file. So that part is working now. But cant seem to get the variable from the container when trying to access from outside.  By this I mean for example I can go into the container and echo the variable:
`
promet@Promets-MBP.attlocal.net: $ docker-compose exec php /bin/bash
root@a41c9229fb88:/var/www/web# echo $BUILD_ENV                                                                                                                                                                                              
dev
`

But when trying to access this variable with the build script the variable is always undefined. So for example if in the build script (or directly from the cmd as well) I do:
`
docker-compose run --rm php echo $BUILD_ENV
`
It will return this:
`
promet@Promets-MBP.attlocal.net: $ ../scripts/drupal-update.sh 
WARNING: The BUILD_ENV variable is not set. Defaulting to a blank string.
Starting simpledockerdrupal_backend_1
`

One work around I found is just taking the variable from the .env file but Im not sure this is what we really want as I suspect we want to read the variable directly from the container. So for example Im able to do this and it does work:
'
source ../.env
build_env=$BUILD_ENV
echo $BUILD_ENV
'
which outputs:
`
promet@Promets-MBP.attlocal.net: $ ../scripts/drupal-update.sh 
dev
`
Any idea what might be the issue?